common/api/metadata.go

```diff
@@ -161,6 +161,10 @@ var (
 		"UpdateTaskQueueConfig":                 {Scope: ScopeNamespace, Access: AccessWrite, Polling: PollingNone},
 		"FetchWorkerConfig":                     {Scope: ScopeNamespace, Access: AccessReadOnly, Polling: PollingNone},
 		"UpdateWorkerConfig":                    {Scope: ScopeNamespace, Access: AccessWrite, Polling: PollingNone},
+		"StartNexusOperation":                   {Scope: ScopeNamespace, Access: AccessWrite, Polling: PollingNone},
+		"RequestCancelNexusOperation":           {Scope: ScopeNamespace, Access: AccessWrite, Polling: PollingNone},
+		"GetNexusOperationInfo":                 {Scope: ScopeNamespace, Access: AccessReadOnly, Polling: PollingNone},
+		"GetNexusOperationResult":               {Scope: ScopeNamespace, Access: AccessReadOnly, Polling: PollingNone},
```

Comment:
Is this PollingCapable since it can result in a long poll? Not sure if we need to mark it as such or not.